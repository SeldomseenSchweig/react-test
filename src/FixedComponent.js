import React from "react";

function FixedComponent(props) {
    console.log(props)

    return (
        <div>

            <p>Hello</p>
            <p>Fixed Component</p>
            <p> {props.favNum}</p>
            <p>{props.favNum +1}</p>
            <p>{props.favNum +2}</p>
        </div>
    )
    
}
FixedComponent.defuaultProps = {
    favNum: 42
};




export default FixedComponent;