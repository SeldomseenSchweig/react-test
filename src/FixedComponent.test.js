import React from 'react';
import ReactDOM from 'react-dom';
import { render } from '@testing-library/react';
import FixedComponent from './FixedComponent'

test("renders without crashing", ()=>{

  render(<FixedComponent />)
})

test("matches snapshot", ()=>{
   const {asFragment} = render(<FixedComponent />);
   expect(asFragment()).toMatchSnapshot();
})