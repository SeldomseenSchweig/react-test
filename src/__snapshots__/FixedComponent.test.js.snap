// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <p>
      Hello
    </p>
    <p>
      Fixed Component
    </p>
    <p>
       
    </p>
    <p />
    <p />
  </div>
</DocumentFragment>
`;
